posts/2017-06-01-may.md

```diff
@@ -0,0 +1,74 @@
+  main:
+    parent: "Posts"
+title: Dev Diary (Mayh 2017)
+author: Corey Montella
+date: 2017-06-1
+---
+
+_(Eve is a new programming language, and this is our development blog. If you’re new to Eve, [start here](http://witheve.com))_
+
+In May we released Preview 3 of v0.3, published some new documentation, and had a meetup.
+
+## Preview 3 of v0.3
+
+### The Eve Syntax Returns
+
+The parser for the Eve syntax returns in Preview 3. While the parser is included again, there’s still no editor bundled, so parsing a program written in the Eve syntax can accomplished in JavaScript like so:
+
+``````javascript
+import {Program} from "witheve";
+
+let program = new Program("program name");
+
+program.load(`
+# An eve program
+
+This is an Eve program
+
+~~~
+search
+  [#person name: "Jeff"]
+
+bind
+  [#ui/text text: "Hi Jeff"]
+~~~`)
+``````
+
+A second method is to download [eve-starter](https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fwitheve%2Feve-starter&sa=D&sntz=1&usg=AFQjCNHEFK6uF3Udjrz51ipbaJv4j6g1BA), and put your *.eve file in the “programs” directory. Then launch the program switcher (run npm start in the eve-start directory, then direct your browser to localhost:8000), and you will see your program listed in the left panel.
+
+We also updated the [example Eve applications](https://github.com/witheve/eve-starter/tree/master/programs) for v0.3. This entailed removing any references to databases and updating records to use the new namespaced tags. We're currently in the process of documenting watchers (e.g. html, svg, events), but if you’re curious how to use these, you can use the example programs as a reference for now.
+
+### Documentation for v0.3
+
+
+As part of this release, we’re also making available our [documentation refresh](http://docs.witheve.com/v0.3/). We’ve updated the layout to match the new Eve website, and we’ve added some summary documents to help new users get oriented in the language. These include:
+
+- A new [quickstart](http://docs.witheve.com/v0.3/tutorials/quickstart.eve/) – We’ve got a new version of the quickstart that aims to be a 5 to 10 minute introduction to Eve. In 6 blocks we introduce blocks, records, and the difference between binding and committing records.
+
+- A new [syntax reference](http://docs.witheve.com/v0.3/syntaxreference/) – This is a 3 page document that acts as a reference for the Eve syntax. You can view it online, or print it out to keep handy on your desk.
+
+- A [library reference](http://docs.witheve.com/v0.3/handbook/libraries/) – This page will act as an index to the standard library. For every function in the standard library, we list its arguments as well as an example demonstrating its usage.
+
+These documents represent the minimum viable documentation to the Eve language. We hope you’ll take a look, and let us know if anything need more clarity or if we’ve omitted any important details. As we update the old docs for v0.3, they'll become available on the v0.3 docs site.
```

Comment:
I still wouldn't talk about "minimum viable documentation" because it reads awkwardly.